tools/main.c

```diff
@@ -60,7 +60,7 @@ main (int   argc,
     case 1:
         execute_man (argv[0], envp);
         LOG_ERR ("failed to load manpage, check your environment / PATH\n");
-        /* fallthrough */
+        exit (1);
```

Comment:
Why not change it to fallsthrough or set up the correct compiler settings in configure.ac?